src/components/Toast/ToastContainer.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import { h, FunctionComponent } from 'preact';
+import { useStore } from '@nanostores/preact';
+import { toastStore } from './toastStore';
+import { Toast } from './Toast';
+
+const DEFAULT_OFFSET = '1.5rem';
+
+export const ToastContainer: FunctionComponent = () => {
+  const toasts = useStore(toastStore);
+
+  return (
+    <div
+      className="flex flex-col items-end fixed"
+      style={{
+        zIndex: 9999,
```

Comment:
Just a heads up that we have a class in [modals](https://github.com/standardnotes/StyleKit/blob/main/src/css/_modal.scss#L9) whose z-index is 10_000. Toasts may need to have higher z-index.